deploy_rln_contract.sh

```diff
@@ -13,7 +13,13 @@ if [ ! -d "waku-rlnv2-contract" ]; then
     git clone https://github.com/waku-org/waku-rlnv2-contract.git
 fi
 
+if [ -z "$RLN_CONTRACT_REPO_COMMIT" ]; then
+    echo "RLN_CONTRACT_REPO_COMMIT is not set"
+    exit 1
+fi
+
 cd /waku-rlnv2-contract
+git checkout $RLN_CONTRACT_REPO_COMMIT
```

Comment:
```suggestion
git checkout $RLN_CONTRACT_REPO_COMMIT
if [ $? -ne 0 ]; then
  echo "Invalid RLN_CONTRACT_REPO_COMMIT set"
  exit 1
fi
```

How about adding this? Up to you.

Reply:
I have verified that failure of the checkout command will exit the script with non zero error code.
It gives an error message like this and exits:
`error: pathspec 'blah-blah' did not match any file(s) known to git`
Therefore there is no need to recheck :)